Description: reduce strength of awk rule so it doesn't get priority over perl scripts
Origin: DSA-2873-1-regression.patch from Debian's file (5.04-5+squeeze5)
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=742262
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=742265

Index: file-5.14/magic/Magdir/commands
===================================================================
--- file-5.14.orig/magic/Magdir/commands	2014-07-10 10:12:10.198525132 -0400
+++ file-5.14/magic/Magdir/commands	2014-07-10 13:15:07.534355056 -0400
@@ -50,6 +50,7 @@
 0	string/wt	#!\ /usr/bin/awk	awk script text executable
 !:mime	text/x-awk
 0	regex/4096	=^\\s{0,100}BEGIN\\s{0,100}[{]	awk script text
+!:strength - 12
 
 # AT&T Bell Labs' Plan 9 shell
 0	string/wt	#!\ /bin/rc	Plan 9 rc shell script text executable
